src/components/AppComponent.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { FC, Fragment, useState, useEffect, useLayoutEffect } from 'react';
```

Comment:
Make sure to rename files with `git mv x/y/oldName x/y/newName`, otherwise `git` will not know it is the same file and diffs and things like `git blame` will not work properly.

Reply:
That makes sense. I've tried fixing it in the latest commit.